src/main/kotlin/no/nav/klage/oppgave/api/controller/SaksbehandlerController.kt

```diff
@@ -30,6 +29,36 @@ class SaksbehandlerController(
         private val logger = getLogger(javaClass.enclosingClass)
     }
 
+
+    @ApiOperation(
+        value = "Hent medunderskriver for en ansatt",
+        notes = "Henter alle medunderskrivere som saksbehandler er knyttet til for en gitt ytelse pg fnr."
+    )
+    @PostMapping(
+        "/medunderskrivere",
```

Comment:
En `POST` på `/medunderskrivere` føles litt rart for å hente ut medunderskrivere?

Reply:
Vi må POSTe når vi behandler fnr dessverre. Men du vil kanskje ha en annen url? Forslag?

Reply:
Blir kanskje mer av en "action" da? `/search/get/medunderskrivere` e.l.?

Reply:
endret til /search/medunderskrivere

Reply:
endret